debug_toolbar/models.py

```diff
@@ -0,0 +1,15 @@
+from django.db import models
+
+
+class DebugToolbarEntry(models.Model):
+    request_id = models.UUIDField(primary_key=True)
+    data = models.JSONField(default=dict)
+    created_at = models.DateTimeField(auto_now_add=True)
+
+    class Meta:
+        verbose_name = "Debug Toolbar Entry"
+        verbose_name_plural = "Debug Toolbar Entries"
```

Comment:
```suggestion
        verbose_name = _("Debug Toolbar Entry")
        verbose_name_plural = _("Debug Toolbar Entries")
```

Not sure about capitalization. I mostly write everything in lowercase.

Reply:
Yes, missed that. I also prefer lowercase.